_posts/2021-05-28-seqan-3-0-3-release.md

```diff
@@ -0,0 +1,133 @@
+
+Furthermore, we see this release as a test run of our way to handle API Stability in the upcoming era of stable releases. We are happy to hear your feedback, so please let us know whether it worked for you and if something could be improved. We believe we found a good system for hinting changes between releases to our users.
+
+But how do we know that we did not miss anything? Our idea is simple: compile our current SeqAn version (3.0.3 in this case) against the tests of our previous Release (3.0.2). If everything compiles and the previous tests pass, we most likely did not break our API.
+
+We counted just two real breaking API changes that can be found in our [changelog](https://docs.seqan.de/seqan/3.0.3/about_changelog.html).
+
+### PacBio Phred Score alphabet
+
+We implemented a new Phred Quality Score alphabet `seqan3::phred94` that represents the full Phred Score range (Sanger format) and is used for PacBio Phred scores of HiFi reads.
+
+### `seqan3::literals` namespace
+
+`seqan3::literals` namespace allows you to import all available seqan3 literal operators at once to simplify code. Of course you can still import individual literal operators.
+
+```cpp
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+
+int main()
+{
+    using namespace seqan3::literals; // Still works: using seqan3::operator''_dna4;
+
+    seqan3::dna4 adenine = 'A'_dna4;
+}
+```
+
+### Member functions in record types
+
+Our records of I/O files have now member functions. We hope that this makes the [documentation of
+records](https://docs.seqan.de/seqan/3.0.3/classseqan3_1_1sequence__record.html) more clear and allows us to add member functions  in the
+future that compute convenient data representation out of the existing data.
+
+One additional aspect is, that this change unifies all our result ranges to have member functions to access the data,
+e.g. [seqan3::alignment_result](https://docs.seqan.de/seqan/3.0.3/classseqan3_1_1alignment__result.html),
+[seqan3::search_result](https://docs.seqan.de/seqan/3.0.3/classseqan3_1_1search__result.html), and
+[seqan3::sequence_record](https://docs.seqan.de/seqan/3.0.3/classseqan3_1_1sequence__record.html).
+
+```cpp
+#include <seqan3/core/debug_stream.hpp>
+#include <seqan3/io/sequence_file/input.hpp>
+
+int main()
+{
+    seqan3::sequence_file_input fin{"my.fastq"};
+
+    for (auto && record: fin)
+    {
+        seqan3::debug_stream << "id: " << record.id() << '\n';
+        seqan3::debug_stream << "sequence: " << record.sequence() << '\n';
+        seqan3::debug_stream << "base_qualities: " << record.base_qualities() << '\n';
+    }
+}
+```
+
+### Other important changes and additions
+
+We put much effort into harmonising our sequence analysis library for the upcoming release 3.1.0. For that we renamed
```

Comment:
```suggestion
We put much effort into harmonising our sequence analysis library for the upcoming 3.1.0 release. For that, we renamed
```

Reply:
Is it `Sequence Analysis Library` or `sequence analysis library`?

Reply:
not sure? let's keep the upper case one.

Reply:
Both are fine, but it was mixed in the post. We should stick to one